examples/Python/Basic/pointcloud.ipynb

```diff
@@ -241,6 +241,37 @@
     "`paint_uniform_color` paints all the points to a uniform color. The color is in RGB space, [0, 1] range."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pointcloud Distance\n",
```

Comment:
`pointcloud` -> `point cloud` to be consistent with the rest.

Reply:
done